doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -162,22 +162,16 @@ systemctl enable wpa_supplicant@${WIFI_INTERFACE}
 # ZSH SHELL #
 #############
 
-# Change user shell to zsh
+# Change user shell to zsh and create a blank config file to prevent user prompt
 chsh -s /usr/bin/zsh ${USER}
+touch ${HOME}/.zshrc
```

Comment:
This isn't needed because `SystemConfiguration` will setup this file correctly. So you should only ever see this prompt once. `SystemConfiguration` also sets the user shell to `zsh`, so you can probably get rid of all of this and just let `SystemConfiguration` handle it all.

Reply:
Yep, you could just leave the install as bash and not install zsh at all during setup. System configuration will install zsh for you and setup the zshrc stuff

Reply:
I think the best thing to do is just use bash until systemconfigurator. This avoids the prompt, and systemconfigurator can set it up properly when it's actually going to be useful.